README.md

```diff
@@ -171,236 +175,3 @@ Options:
   --address <value>  Centrifuge asset store contract address (default: "0xc279648CE5cAa25B9bA753dAb0Dfef44A069BaF4")
 ```
 
-# ChainBridge-Solidity Data Layout
```

Comment:
Do you think it might be useful to put the call data location of the data being passed in into the notion page as well? Or would that be too much? I think it would help with folks making sense of the calldatacopy operations.

Reply:
Included here: https://www.notion.so/chainsafe/ChainBridge-Solidity-ad0b0e53e5204e7c8e5e850cbd40392b